include/libcyphal/transport/transfer_id_map.hpp

```diff
@@ -0,0 +1,228 @@
+/// @copyright
```

Comment:
JFYI, this is not a new file, but rename and a bit modification of the above `transfer_id_generators.hpp`